### DISCUSSION TITLE
Send value to Meter widget

### DISCUSSION THREAD

**Discussion by CodingLama** [israel-dryer/ttkbootstrap#574] [Q&A]
How do I send a value to the Meter widget? It doesn't have a `variable` or `textvariable` attribute, so how one can influence the behaviour of this widget from the outside?

**Comment by TheDemonGuard** [comment]
For example, if it is required to reflect the value in the "text field" to the meter,
it is possible by registering the "amountusedvar" variable as follows:

`entry = ttk.Entry(textvariable=meter.amountusedvar)`
(NOTE: you need to make sure the value entered in the "text field" is a number, otherwise it will throw the following exception,
"_tkinter.TclError: expected floating-point number")

For more example, please refer to the following documentation.
[https://ttkbootstrap.readthedocs.io/en/latest/api/widgets/meter/#:~:text=Examples](https://ttkbootstrap.readthedocs.io/en/latest/api/widgets/meter/#:~:text=Examples)
